### ISSUE TITLE
Cannot read properly 'wait"

### ISSUE THREAD

**Issue by thevadimspivak** [ipfs-shipyard/nft-school-examples#3]
Hello. I was trying to do end-to-end lesson. 
When I was trying to mint my NFT, I found out a mistake, can you help me to fix this error?
This looks like this
![image](https://user-images.githubusercontent.com/63224089/126216920-919dda66-9439-406a-8ea7-a1a8c82f726b.png)
Thank you.

**Comment by jessicaschilling** [comment]
@yusefnapora @johndmulhausen are you able to have a look?

**Comment by yusefnapora** [comment]
Hi @thevadimspivak thanks for the report - I should be checking for null there!

Did you add any test Ether to your account before minting? If not, that may be the cause of the failed transaction, and you could try again after getting some play Eth from the faucet.

**Comment by thevadimspivak** [comment]
Yes, I did. I have a few test Ether on balance, but it does not help at all.
Any time I try to make any transaction I got this message.
![image](https://user-images.githubusercontent.com/63224089/126226788-874b2555-facf-4571-b501-a76d1534985a.png)
Thank you for the answer.

**Comment by yusefnapora** [comment]
Thanks for the error output. I'll try to reproduce this tomorrow and debug.

**Comment by thevadimspivak** [comment]
Did you manage to fix it?

**Comment by MuhdRaffiq** [comment]
Did anyone manage to get the issue here? I receive the same error as well for wait()

It was doing good for the local chain. But when I use the code to connect to an NFT smart contracts in BSC test it produces this error. 

I have:
1. test BNB for the execution of the contract
2. update BSC testnet network at NETWORKS in constant.js
3. put testnet at targetNetwork in app.jsx
4. put the contract address in contracts/NFTMinter.address
5. put the contracts address ABI at NFTMinter.abi in contracts
6. put the bytecode address at NFTMinter.bytecode in contracts
4. Reset account my metamask wallet 

I tried looking into the code, cant seem to find any problem because its working for my local chain

**Comment by MuhdRaffiq** [comment]
> Did anyone manage to get the issue here? I receive the same error as well for wait()
> 
> It was doing good for the local chain. But when I use the code to connect to an NFT smart contracts in BSC test it produces this error.
> 
> I have:
> 
> 1. test BNB for the execution of the contract
> 2. update BSC testnet network at NETWORKS in constant.js
> 3. put testnet at targetNetwork in app.jsx
> 4. put the contract address in contracts/NFTMinter.address
> 5. put the contracts address ABI at NFTMinter.abi in contracts
> 6. put the bytecode address at NFTMinter.bytecode in contracts
> 7. Reset account my metamask wallet
> 
> I tried looking into the code, cant seem to find any problem because its working for my local chain

nevermind. I solved it. I got it wrong on function input

**Comment by hacktar** [comment]
> > Did anyone manage to get the issue here? I receive the same error as well for wait()
> > It was doing good for the local chain. But when I use the code to connect to an NFT smart contracts in BSC test it produces this error.
> > I have:
> > 
> > 1. test BNB for the execution of the contract
> > 2. update BSC testnet network at NETWORKS in constant.js
> > 3. put testnet at targetNetwork in app.jsx
> > 4. put the contract address in contracts/NFTMinter.address
> > 5. put the contracts address ABI at NFTMinter.abi in contracts
> > 6. put the bytecode address at NFTMinter.bytecode in contracts
> > 7. Reset account my metamask wallet
> > 
> > I tried looking into the code, cant seem to find any problem because its working for my local chain
> 
> nevermind. I solved it. I got it wrong on function input

Sorry, how did you solve ? Thanks in advance

**Comment by MuhdRaffiq** [comment]
My mistake was that I have put the wrong input for the function's parameter.

**Comment by tony123S** [comment]
I got this issue because of the gas fee higher than the wallet balance
